features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ThrottlingPolicies/BusinessPlans.js

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, {Component} from 'react'
+
+import {Link} from 'react-router-dom'
+import Grid from 'material-ui/Grid';
+import Paper from 'material-ui/Paper';
+import Typography from 'material-ui/Typography';
+import Table, { TableBody, TableCell, TableHead, TableRow } from 'material-ui/Table';
+import AppBar from 'material-ui/AppBar';
+import Toolbar from 'material-ui/Toolbar';;
+import IconButton from 'material-ui/IconButton';
+import Divider from 'material-ui/Divider';
+import Button from 'material-ui/Button';
+import MenuIcon from 'material-ui-icons/Menu';
+import TextField from 'material-ui/TextField';
+
+import API from '../../data/api'
+
+export default class BusinessPlans extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            policies: null,
+            selectedRowKeys: [],
+            open:false,
+            message: ''
+        };
+
+    }
+    componentDidMount() {
+        const api = new API();
```

Comment:
Can you attach a screen shot of the rendered UI?

Reply:
Attached the page. Please note that background is in Gray colour because there are still ant components in the admin ui. Once they are changed to material-ui, background and the other menu items will be changed.

Reply:
noted